### ISSUE TITLE
`exists` Keyword not Working on AArch64 Linux

### ISSUE THREAD

**Issue by redraiment** [duckdb/duckdb#14389]
### What happens?

I downloaded duckdb_cli-linux-aarch64.zip from https://github.com/duckdb/duckdb/releases/tag/v1.1.2, and then executed the following SQL query multiple times:

```sql
select count(*) from range(1000, 3000) as a(v), range(1000, 3000) as b(v) where exists (select a.v != b.v);
```

Each time the query resulted in different outcomes, as shown in the screenshots below:

![image](https://github.com/user-attachments/assets/273dba63-1e8b-4bf8-81e0-1a4367bad0c9)

I have only encountered this issue on the AArch64 version of the *Linux* system; testing on the AArch64 version of *macOS* did not reveal any problems.

### To Reproduce

```sql
select count(*) from range(1000, 3000) as a(v), range(1000, 3000) as b(v) where exists (select a.v != b.v)
```

### OS:

AArch64 Debian GNU/Linux 12 (bookworm)

### DuckDB Version:

1.1.2

### DuckDB Client:

duckdb_cli

### Full Name:

Zepeng, Zhang

### Affiliation:

Yinfn.com

### What is the latest build you tested with? If possible, we recommend testing with the latest nightly build.

I have tested with a stable release

### Did you include all relevant data sets for reproducing the issue?

Yes

### Did you include all code required to reproduce the issue?

- [X] Yes, I have

### Did you include all relevant configuration (e.g., CPU architecture, Python version, Linux distribution) to reproduce the issue?

- [X] Yes, I have

**Comment by redraiment** [comment]
Add an additional piece of information:

I installed version v1.1.2 of duckdb on an AArch64 Android device through Termux with the command `pkg install duckdb`, and it returned the correct results.

Maybe only the one on Github Releases page with the issue.

**Comment by szarnyasg** [comment]
Thanks, I could reproduce with Ubuntu 24.04 running a Docker container on macOS (Apple Silicon M2).

**Comment by szarnyasg** [comment]
Hi @redraiment, a status report: we have found the cause of this issue and plan to fix it in DuckDB v1.1.3, scheduled for next week: https://duckdb.org/docs/dev/release_calendar.html
